app/services/criteria_matcher.py

```
@@ -361,7 +361,7 @@ def _get_macos_last_open_time(file_path: Path) -> Optional[float]:
 
             # Method 2: Try xattr (fallback)
             xattr_result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)